l10n_es_account_summary/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo - Account summary
+#    Copyright (C) 2014 Luis Martinez Ontalba (www.tecnisagra.com).
+#      
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import account_summary
```

Comment:
Corregido

El 05/01/2015 a las 12:01, Pedro M. Baeza escribió:
>
> In l10n_es_account_summary/__init__.py 
> <https://github.com/OCA/l10n-spain/pull/84#discussion-diff-22455650>:
>
> > +#    This program is free software: you can redistribute it and/or modify
> > +#    it under the terms of the GNU Affero General Public License as published
> > +#    by the Free Software Foundation, either version 3 of the License, or
> > +#    (at your option) any later version.
> > +#
> > +#    This program is distributed in the hope that it will be useful,
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#    GNU Affero General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU Affero General Public License
> > +#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +##############################################################################
> > +
> > +import account_summary
>
> |from . import account_summary|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/l10n-spain/pull/84/files#r22455650>.
>